src/components/Banner/Banner.mdx

```diff
@@ -0,0 +1,236 @@
+colors used in a specific variant.
+
+**IMPORTANT:** The text color for links within the content will NOT use the
+DS standard link color. Instead, the color will match the color of the content.
+Additionally, links will always be underlined.
+
+**IMPORTANT:** For the "negative" variant only, the heading and text color will
+both be set to the same color, `ui.error.primary` and `dark.ui.error.primary`
+for dark mode.
+
+<Canvas of={BannerStories.TypeVariants} />
+
+## Banner Heading
+
+The `Banner` component can be rendered with and without a heading. By
+default, when a title is passed to the `heading` prop, the
+`Banner` component will render an `h2` element sized as a `heading6`
+heading. If this causes accessibility issues in your app, you can pass in a
+custom `Heading` element.
+
+In the following examples, the `Banner` component will render custom `h3`
+and `h5` elements. Note that no matter what heading level is set, the size will
+always be set to `heading6` internally by the `Banner` component.
+
+<Source
+  code={`
+
+const customH3 = <Heading level="h3">Custom H3 Heading</Heading>;
+const customH5 = <Heading level="h5">Custom H5 Heading</Heading>;
+
+<Banner heading={customH3} {...props} />
+<Banner heading={customH5} {...props} />
+`} language="tsx" />
+
+<Canvas of={BannerStories.CustomBannerHeading} />
+
+## Banner Icon
+
+The default icon can be overridden by using the `icon` prop to pass a
+custom `Icon` component.
+
+<Source
+  code={`
+<Banner
+  content={
+    <>
+      Cras mattis consectetur purus sit amet fermentum. Maecenas faucibus
+      mollis interdum. Morbi leo risus, porta ac consectetur ac,
+      vestibulum at eros. Cum sociis natoque penatibus et magnis dis
+      parturient montes, nascetur ridiculus mus.
+    </>
+  }
+  heading="Banner with Custom Icon"
+  icon={
+    <Icon
+      name="actionLightbulb"
+      title="Banner with custom icon"
+      size="large"
+    />
+  }
+  type="informative"
+/>
+`}
+  language="tsx"
+/>
+
+<Canvas of={BannerStories.CustomBannerIcon} />
+
+## With HTML Content
+
+The `content` prop can accept HTML.
+
+<Source
+  code={`
+<Banner
+  heading="Standard Banner with HTML content"
+  content={
+    <>
+      <Text>
+        Cras mattis consectetur purus sit amet fermentum. Maecenas
+        faucibus mollis interdum.
+      </Text>
+      <Text noSpace>
+        Morbi leo risus, porta ac consectetur ac, vestibulum at eros.{" "}
+        <b>
+          Cum sociis natoque penatibus et magnis dis parturient montes,
+          nascetur ridiculus mus
+        </b>
+        . <Link href="#">This is a link</Link>.
+      </Text>
+    </>
+  }
+/>
+`}
+  language="tsx"
+/>
+
+<Canvas of={BannerStories.HTMLContent} />
+
+## Custom Color
+
+It is recommended to rely on the color styles set with the `type` prop.
+However, in cases where a different color is necessary, a value from a defined
+set of colors can be used.
```

Comment:
Include something about the colors being used in tandem.  When one is set, the other is required.